packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1044,7 +1044,7 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 	 * @see
 	 * - [The reserved entry `slug` field](https://docs.astro.build/en/guides/content-collections/)
 	 * @description
-	 * A content collection schema should not contain the `slug` field. This is reserved by Astro for generating entry slugs. Remove the `slug` field from your schema, or choose a different name.
+	 * A content collection schema should not contain the `slug` field. Remove `slug` from your schema in order to use custom slugs in your frontmatter.
```

Comment:
```suggestion
	 * A content collection schema should not contain the `slug` field. Remove `slug` from your schema. You can still use custom slugs in your frontmatter.
```
I realized after our discussion that to be a little more generic (i.e. maybe some people are getting this error who are NOT trying to use custom slugs, but who think they *need* to specify `slug` when they don't), we could use something like this. What do you think?